### DISCUSSION TITLE
Inline highlighting to highlight stuff like placeholder values

### DISCUSSION THREAD

**Discussion by dbrian57** [PrismJS/prism#3523] [Q&A]
Hi, is there a way to use Prism to highlight not just specific lines, but specific text in a line? For example, we'd love to be able to highlight placeholder values in code that users need to replace on their own, such as the value for the `email:` field here:

```
{{< prism title="cert-manager-issuer.yaml" language="YAML" >}}
...
spec:
  acme:
    email: <YOUR_VALID_EMAIL_ADDRESS_HERE>
    server: https://acme-v02.api.letsencrypt.org/directory
...
{{< /prism >}}
```

Would love to know. Thanks!

**Comment by RunDevelopment** [comment]
You could use the [Keep Markup plugin](https://prismjs.com/plugins/keep-markup/) to do something like this:

```html
spec:
  acme:
    email: <mark>&lt;YOUR_VALID_EMAIL_ADDRESS_HERE&gt;</mark>
    server: https://acme-v02.api.letsencrypt.org/directory
```

and give the `mark` element special styles via CSS. 

I only used `mark` as an example. Keep Markup will keep any tags.
